docs/modules/ROOT/pages/how-to/use-github-actions.adoc

```diff
@@ -0,0 +1,228 @@
+
+. Clone the project:
++
+[source,shell]
+--
+git clone git@github.com:[YOUR_GITHUB_USERNAME]/fortune-go.git
+--
+
+. `cd` into it:
++
+[source,shell]
+--
+cd fortune-go
+--
+
+. Since the code has been imported from another repository, GitHub Actions aren't automatically available. Enable them by going to the menu:Settings[Actions > General] screen and selecting the "Allow all actions" option.
+
+=== 3. Add a Deployment File
+
+Add a file named `deployment.yaml` at the root of the project with the following information:
+
+[source,yaml]
+--
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  name: fortune-go
+spec:
+  replicas: 1
+  template:
+    spec:
+      containers: # <1>
+      - image: registry.[YOUR_CHOSEN_ZONE].appuio.cloud/[YOUR_USERNAME]-fortune-go/fortune-go:latest
+        imagePullPolicy: Always
+        name: fortune-container
+        ports:
+        - containerPort: 8080
+    metadata:
+      labels:
+        app: fortune-go
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: fortune-go
+spec:
+  ports:
+    - port: 8080
+      targetPort: 8080
+  selector:
+    app: fortune-go
+  type: ClusterIP
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-production
+  name: fortune-go-ingress
+spec:
+  rules:
+  - host: [YOUR_USERNAME]-fortune-go.apps.[YOUR_CHOSEN_ZONE].appuio.cloud # <1>
+    http:
+      paths:
+      - pathType: Prefix
+        path: /
+        backend:
+          service:
+            name: fortune-go
+            port:
+              number: 8080
+  tls:
+  - hosts:
+    - [YOUR_USERNAME]-fortune-go.apps.[YOUR_CHOSEN_ZONE].appuio.cloud
+    secretName: fortune-go-cert
+--
+<1> Remember to customize the parts marked as `[YOUR_USERNAME]` and `[YOUR_CHOSEN_ZONE]` to your liking (and according to the xref:references/zones.adoc[Zones documentation page]).
+
+include::partial$url-length-warning.adoc[]
+
+=== 4. Configure Secrets
+
+To be able to communicate with {product}, GitHub Actions use secrets. And to connect GitHub Actions to {product} we're going to use a dedicated service account.
+
+. Create a service account
++
+[source,shell]
+--
+oc create sa github-actions
+--
+
+. Add the proper role to the service account:
++
+[source,shell]
+--
+oc adm policy add-role-to-user edit -z github-actions
+--
+
+. On your laptop, copy the token of the service account:
++
+[source,shell]
+--
+oc sa get-token github-actions
+--
+
+. On GitHub, select the menu:Settings[Secrets > Actions] screen and click on the btn:[New repository secret] button.
+. Create the following secrets:
++
+.. `REGISTRY_USERNAME`: Should be `github-actions`.
+.. `REGISTRY_PASSWORD`: The token you copied in the previous step, corresponding to the `github-actions` service account.
+.. `REGISTRY_URL`: The URL to the "Integrated Cluster Registry Domain" in your chosen {product} zone, as defined in the xref:references/zones.adoc[Zones documentation page].
```

Comment:
Maybe we should already link to http://portal.appuio.cloud/zones. If you leave it like that, please mention it at https://ticket.vshn.net/browse/APUB-233, so we can change it as part of that issue.

Reply:
Changed to https://portal.appuio.cloud/zones